src/data.js

```diff
@@ -2064,6 +2064,32 @@ const pages = [
       'YouTuber',
     ],
   },
+  {
+    name: 'Michał Miszczyszyn',
+    description:
+      'Motivated full-stack developer not afraid to use any technology. Experienced developer and leader. He, him.',
+    url: 'https://typeofweb.com/wspolpraca/',
```

Comment:
I've checked this out and despite my poor Polish 🇵🇱 I don't believe it's a "uses" page, it's more of a CV I believe, let me know if I'm wrong.

`awesome-uses` needs submissions to be "uses" pages.

Reply:
@HugoDF You're right, I updated the file again here: https://github.com/wesbos/awesome-uses/pull/388
Sorry for trouble!